### ISSUE TITLE
Validation loss doesn't change over the epochs

### ISSUE THREAD

**Issue by ploe24** [perslev/U-Time#43]
Even after hundreds of epochs, the loss is still the same value

Just a few examples (loss always 0.821 and never changes):
[training.txt](https://github.com/perslev/U-Time/files/7055905/training.txt)

**Comment by perslev** [comment]
Hi ploe24

I am sorry that I never returned to you on this, I simply forgot. There was a bug in the validation loss computation in a subset of earlier versions, but this has been fixed since 875ca86550fe54dfd653a0e4c59f071ce34ce106 (i.e., the latest version on branch `v0.0.2`). Could you try and pull the latest version e.g., on `master` and confirm that it works for you now? 

Cheers Mathias
